runtime/autoload/remote/host.vim

```diff
@@ -118,18 +118,26 @@ function! remote#host#RegisterPlugin(host, path, specs) abort
 endfunction
 
 
-function! s:GetManifest() abort
+function! s:GetInitScript() abort
   let prefix = exists('$MYVIMRC')
         \ ? $MYVIMRC
         \ : matchstr(get(split(capture('scriptnames'), '\n'), 0, ''), '\f\+$')
-  return fnamemodify(expand(prefix, 1), ':h')
+  if empty(prefix) || fnamemodify(prefix, ':p') =~# '^'.$VIMRUNTIME
+    if exists('$XDG_CONFIG_HOME') && isdirectory($XDG_CONFIG_HOME)
+      let prefix = $XDG_CONFIG_HOME.'/nvim/init.vim'
+    else
+      let prefix = '~/nvim'
+    endif
+  endif
+  let v:rplugin_init = fnamemodify(expand(prefix, 1), ':h')
         \.'/.'.fnamemodify(prefix, ':t').'-rplugin~'
```

Comment:
> I like that more. I could also have it rename the old file if it exists unless you can think of a reason that would be bad.

That would be slick, but I'm ok with just mentioning it at https://github.com/neovim/neovim/wiki/Following-HEAD